src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -564,647 +556,613 @@ namespace HVACUnitaryBypassVAV {
-                    ShowContinueError(state, format(" Occurs in {} = {}", CurrentModuleObject, CBVAV(CBVAVNum).Name));
-                    CBVAV(CBVAVNum).MaxHeatAirVolFlow = CBVAV(CBVAVNum).FanVolFlow;
+                    ShowContinueError(state, format(" Occurs in {} = {}", CurrentModuleObject, thisCBVAV.Name));
+                    thisCBVAV.MaxHeatAirVolFlow = thisCBVAV.FanVolFlow;
                 }
             }
 
             //   only check that OA flow in cooling is >= SA flow in cooling when they are not autosized
-            if (CBVAV(CBVAVNum).CoolOutAirVolFlow > CBVAV(CBVAVNum).MaxCoolAirVolFlow && CBVAV(CBVAVNum).CoolOutAirVolFlow != DataSizing::AutoSize &&
-                CBVAV(CBVAVNum).MaxCoolAirVolFlow != DataSizing::AutoSize) {
+            if (thisCBVAV.CoolOutAirVolFlow > thisCBVAV.MaxCoolAirVolFlow && thisCBVAV.CoolOutAirVolFlow != DataSizing::AutoSize &&
+                thisCBVAV.MaxCoolAirVolFlow != DataSizing::AutoSize) {
                 ShowWarningError(state, format("{}: {} cannot be greater than {}", CurrentModuleObject, cNumericFields(4), cNumericFields(1)));
                 ShowContinueError(state, format(" {} is reset to the fan flow rate and the simulation continues.", cNumericFields(4)));
-                ShowContinueError(state, format("Occurs in {} = {}", CurrentModuleObject, CBVAV(CBVAVNum).Name));
-                CBVAV(CBVAVNum).CoolOutAirVolFlow = CBVAV(CBVAVNum).FanVolFlow;
+                ShowContinueError(state, format("Occurs in {} = {}", CurrentModuleObject, thisCBVAV.Name));
+                thisCBVAV.CoolOutAirVolFlow = thisCBVAV.FanVolFlow;
             }
 
             //   only check that SA flow in heating is >= OA flow in heating when they are not autosized
-            if (CBVAV(CBVAVNum).HeatOutAirVolFlow > CBVAV(CBVAVNum).MaxHeatAirVolFlow && CBVAV(CBVAVNum).HeatOutAirVolFlow != DataSizing::AutoSize &&
-                CBVAV(CBVAVNum).MaxHeatAirVolFlow != DataSizing::AutoSize) {
+            if (thisCBVAV.HeatOutAirVolFlow > thisCBVAV.MaxHeatAirVolFlow && thisCBVAV.HeatOutAirVolFlow != DataSizing::AutoSize &&
+                thisCBVAV.MaxHeatAirVolFlow != DataSizing::AutoSize) {
                 ShowWarningError(state, format("{}: {} cannot be greater than {}", CurrentModuleObject, cNumericFields(5), cNumericFields(2)));
                 ShowContinueError(state, format(" {} is reset to the fan flow rate and the simulation continues.", cNumericFields(5)));
-                ShowContinueError(state, format("Occurs in {} = {}", CurrentModuleObject, CBVAV(CBVAVNum).Name));
-                CBVAV(CBVAVNum).HeatOutAirVolFlow = CBVAV(CBVAVNum).FanVolFlow;
+                ShowContinueError(state, format("Occurs in {} = {}", CurrentModuleObject, thisCBVAV.Name));
+                thisCBVAV.HeatOutAirVolFlow = thisCBVAV.FanVolFlow;
             }
 
             if (UtilityRoutines::SameString(Alphas(14), "Coil:Cooling:DX:SingleSpeed") ||
```

Comment:
This should also probably be an enumeration with `getEnumerationValue` because you are uppercasing stuff all over the place here, as well as doing the same string comparisons multiple times.